ci/jenkins/utils.sh

```diff
@@ -14,6 +14,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+function check_and_cleanup_buildkit_cache() {
```

Comment:
should we replace `buildkit_cache` with `docker_build_cache` in the function name?